metals/src/main/scala/scala/meta/internal/metals/doctor/ProblemResolver.scala

```diff
@@ -227,7 +227,7 @@ class ProblemResolver(
 
     // 1.0.0-M3 or higher is valid
     def outdatedMunitInterface =
-      if (!isTestExplorerProvider()) None
+      if (!isTestExplorerProvider() || scalaTarget.isSbt) None
```

Comment:
This is a great change. I've actually had a few people ask about this since they think something is wrong when they see warnings about not being able to debug targets when it's just an sbt target anyways.

Reply:
I think I've only seen people with JUnit problem but I was too lazy to check what was the cause and I took the easier path to add this to the condition. It still bothers me, but not that much to spend time on it. it's better to do sth more useful :D